GPU/Common/TextureCacheCommon.cpp

```diff
@@ -1407,7 +1407,8 @@ bool TextureCacheCommon::CheckFullHash(TexCacheEntry *const entry, bool &doDelet
 				} else {
 					secondKey = entry->fullhash | ((u64)entry->cluthash << 32);
 					secondCacheSizeEstimate_ += EstimateTexMemoryUsage(entry);
-					secondCache_[secondKey] = *entry;
+					// Is this wise? We simply copy the entry.
```

Comment:
Well, this is why it has doDelete as false, but at the time moving the pointer wasn't really an option.

Basically the goal is to keep the old texture details, as well as the texture object (ptr/name/whatever) and keep it in this secondary cache (filling VRAM like crazy, maybe.)  If that hash happens to get used again, we'll use this already-uploaded texture that we never deleted.

So I think a "branch" of the structure like this is fine.

-[Unknown]